autodoc.pl

```diff
@@ -498,13 +498,26 @@ ($$)
 
     my $file_is_C = $file =~ / \. [ch] $ /x;
 
-    # Count lines easier
-    my $get_next_line = sub { $line_num++; return <$fh> };
+    # Count lines easier and handle apidoc continuation lines
+    my $get_next_line = sub {
+            my $contents = <$fh>;
+            return unless defined $contents;
+
+            $line_num++;
+            while ($contents =~ s/ ^ ( =for \ apidoc .*) \s* \\ \n /$1/x) {
```

Comment:
This allows only a single space between `=for` and `apidoc`, but the match in the loop below allows 1 or multiple.

This line should remove the `\n` but you also chomp below.

It might be good to allow extra whitespace between the `\\` and the newline to avoid trailing whitespace looking like it should continue to a person reading the code, while `autodoc.pl` doesn't treat it as one.

Reply:
These points have made me close this request, and integrate  the relevant portions into some commits in  #22377